app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -189,15 +197,15 @@ def create_new_entries
     alias create_from_worktype create_new_entries
     alias create_from_all create_new_entries
 
-    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
the changes from lines 200-237 were done in https://github.com/samvera-labs/bulkrax/pull/569. if you merge in main to this branch, this diff should go away.

Reply:
That sounds good i'll take a look